hack/testing-olm-upgrade/test-030-olm-upgrade-n-1-n.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+# Given an OLM manifest, verify a green field deployment
+# of elasticsearch by asserting EO creates and upgrades
+# the resources that beget the operands that make up logging.
+
+set -euo pipefail
+
+repo_dir="$( cd "$(dirname "$0")/../.." ; pwd -P )"
+source "$repo_dir/hack/testing-olm/utils"
+source "$repo_dir/hack/testing-olm/assertions"
+
+start_seconds=$(date +%s)
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-"$repo_dir/_output/"}
+ARTIFACT_DIR="${ARTIFACT_DIR}/$(basename ${BASH_SOURCE[0]})"
+if [ ! -d $ARTIFACT_DIR ] ; then
+  mkdir -p $ARTIFACT_DIR
+fi
+
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+TIMEOUT_MIN=$((2 * $minute))
+ES_POD_TIMEOUT=$((5 * $minute))
+manifest_dir=${repo_dir}/manifests
```

Comment:
IIRC, the single-source of truth manifest dir will be `bundle/manifests` in CPAAS right? If yes, we should safeguard ourselves already for that here.